app/script/conversation/ConversationRepository.coffee

```diff
@@ -133,11 +134,20 @@ class z.conversation.ConversationRepository
     queue_entry = @sending_queue()[0]
     if queue_entry
       @sending_blocked = true
+      @sending_interval = window.setInterval =>
+        return if @conversation_service.client.request_queue_blocked()
+        @logger.log @logger.levels.ERROR, 'Sending of message from queue failed, unblocking queue', {entry_function: @sending_queue()[0].function, queue: @sending_queue()}
+        @sending_blocked = false
+        @_execute_from_sending_queue()
+        window.clearInterval @sending_interval
+      , 60000
```

Comment:
Somewhere in https://github.com/wireapp/wire-webapp/blob/dev/app/script/config.coffee would be better I would say :/